doc/api/index.rst

```diff
@@ -4,41 +4,69 @@ API Reference
 Matplotlib interfaces
 ---------------------
 
-Matplotlib provides two different interfaces:
-
-- **Axes interface**: create a `.Figure` and one or more `~.axes.Axes` objects
-  (typically using `.pyplot.subplots`), then *explicitly* use methods on these objects
-  to add data, configure limits, set labels etc.
-- **pyplot interface**: consists of functions in the `.pyplot` module. Figure and Axes
-  are manipulated through these functions and are only *implicitly* present in the
-  background.
-
-See :ref:`api_interfaces` for a more detailed description of both and their recommended
-use cases.
+Matplotlib has two interfaces. See :ref:`api_interfaces` for a more detailed
+description of both and their recommended use cases.
 
 .. grid:: 1 1 2 2
-    :padding: 0 0 1 1
+    :padding: 0
+    :gutter: 2
 
     .. grid-item-card::
+        :shadow: none
+        :class-footer: api-interface-footer
 
         **Axes interface** (object-based, explicit)
-        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+        create a `.Figure` and one or more `~.axes.Axes` objects, then *explicitly* use
+        methods on these objects to add data, configure limits, set labels etc.
+
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
         API:
 
         - `~.pyplot.subplots`: create Figure and Axes
         - :mod:`~matplotlib.axes`: add data, limits, labels etc.
         - `.Figure`: for figure-level methods
 
+        +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+        Example:
```

Comment:
Slight preference to keep them. It's more explicit what the sections are about, in particular if you have only a quick glance at it. I think the cards are still compact enough so that we can afford the extra vertical space.